frontend/app/src/components/helper/BinancePairsSelector.vue

```diff
@@ -1,70 +1,68 @@
 <template>
-  <v-card v-bind="$attrs">
-    <div class="mx-4 pt-2">
-      <v-autocomplete
-        :items="allMakerts"
-        :filter="filter"
-        :search-input.sync="search"
-        multiple
-        :loading="loading"
-        :disabled="loading"
-        hide-details
-        hide-selected
-        hide-no-data
-        return-object
-        chips
-        clearable
-        :outlined="outlined"
-        :open-on-clear="false"
-        :label="label ? label : $t('binance_market_selector.default_label')"
-        :class="outlined ? 'binance-market-selector--outlined' : null"
-        item-text="address"
-        item-value="address"
-        class="binance-market-selector"
-        :value="selection"
-        @input="input($event)"
-        @change="search = ''"
+  <v-autocomplete
+    v-bind="$attrs"
+    :items="allMakerts"
```

Comment:
Can we change this to `allMarkets`, it seems I missed that when we initially added the pair support